src/main.js

```diff
@@ -342,6 +342,20 @@ app.on('ready', () => {
     console.log('The application has crashed.');
   });
 
+  // Add Alt+Cmd+(Right|Left) as alternative to switch between servers
```

Comment:
@wvds Would you move this part into `src/main/mainWindow.js`? It would make sense when adding further hidden shortcuts.

Reply:
I've updated the pull request :)